util/src/main/java/io/kubernetes/client/util/SSLUtils.java

```diff
@@ -316,7 +317,13 @@ private static void loadDefaultKeyStoreFile(KeyStore keyStore, char[] keyStorePa
   private static boolean loadDefaultStoreFile(KeyStore keyStore, File fileToLoad, char[] passphrase)
       throws CertificateException, NoSuchAlgorithmException, IOException {
     if (fileToLoad.exists() && fileToLoad.isFile() && fileToLoad.length() > 0) {
-      keyStore.load(new FileInputStream(fileToLoad), passphrase);
+      FileInputStream inputStream = null;
```

Comment:
Maybe use the Java8 syntax:

```java
try (FileInputStream inputStream = new File InputStream(fileToLoad)) {
    ....
}
```

Reply:
oh, I guess you're not just calling `close()` nevermind.

Reply:
Make sense to use Java8 closing.